modules/dbnd-airflow/src/dbnd_airflow/functional/dbnd_cmd_operators.py

```diff
@@ -1,10 +1,11 @@
-def dbnd_task_as_bash_operator(task_cls, name=None, cmd_line=None, **kwargs):
+def dbnd_task_as_bash_operator(dbnd_func_proxy, name=None, cmd_line=None, **kwargs):
```

Comment:
it's task_cls, I think we need to make sure we can do task_definition on proxy itself. , let's put this bugfix on hold